client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -350,11 +354,12 @@ function useAssignablePaymentMethods() {
 	const existingCardMethods = useCreateExistingCards( {
 		storedCards,
 		stripeConfiguration,
+		activePayButtonText: translate( 'Use this card' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Use this card', { context: 'Button label'} )` *ES Score: 12*